python/python/ert/ecl/fortio.py

```diff
@@ -184,7 +184,7 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.__fortio.close()
-        return False
+        return exc_type is not None
```

Comment:
That could be, do we have that many context managers?

The most important part is to _not_ return `True` always, I don't know exactly what the bad consequences are of always returning `False`, but I think it means that _"something bad happened and we didn't deal with it"_.